doc/source/index.rst

```diff
@@ -1,16 +1,26 @@
 SpiNNaker Manchester (PyNN 0.9)
 =============================
-These pages document the main Manchester python code for the PyNN 0.9 branch of the SpiNNaker_ Project which can be found on github_
-
-Alternative way to run SpiNNaker_ are using the SpiNNakerGraphFrontEnd_.
-
+These pages document sPyNNaker 6, an implementation of PyNN_ 0.9 for the
```

Comment:
sPyNNaker 6 ????

Reply:
Do we actually still need to say it is PyNN 0.9 ?

Reply:
I don't really care too much either way.